R/pugixml.R

```diff
@@ -54,6 +56,25 @@ read_xml <- function(xml, pointer = TRUE, escapes = FALSE, declaration = FALSE,
   if (identical(xml, ""))
     xml <- "<NA_character_ />"
 
+  # Clean xml files from x namespace. Otherwise all nodes might look like
+  # <x:node/> and not <node/>. https://github.com/JanMarvin/openxlsx2/pull/213
+  has_x_ns <- isTRUE(options()[["openxlsx2.has_x_namespace"]])
```

Comment:
`getOption()` instead

would `options(openxlsx2.namespace_x = TRUE)` make sense?

Reply:
Thanks wasn't aware of `getOption()`, will add this in #399